### DISCUSSION TITLE
Is there an xs size button?

### DISCUSSION THREAD

**Discussion by OmranAbazid** [carbon-design-system/carbon#14659] [Help] [answered]
I noticed that when i try using `size="xs"` on button, it makes button slightly smaller than the "sm" size. 
So do the carbon button support "xs" size because the typescript rejects it currently

**Comment by tw15egan** [comment, marked as answer]
No, `size="xs"` is not currently supported, unfortunately.

https://carbondesignsystem.com/components/button/usage/#button-sizes
